src/Queue/SqsQueue.php

```diff
@@ -29,15 +29,15 @@ class SqsQueue extends AbstractQueue implements SqsQueueInterface
     /**
      * Constructor
      *
-     * @param SqsClient        $sqsClient
-     * @param SqsQueueOptions  $options
-     * @param string           $name
+     * @param SqsClient $sqsClient
```

Comment:
You can simply remove the PHPDoc if the arguments in the function are allready typed :scissors:.